Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
```

Comment:
`yum install` refreshes the metadata, and you can force it with `--refresh`.
Running `yum update` is not a problem, just something I feel most of the times we leave up to the base image.

If we run a docker build using the cache, it won't "update" packages anyway.

It also creates a layer that is potentially larger than it needs to be: we'd expect this image to be centos + (nagios + related deps), but we end up with centos + (nagios + related deps + updates for unrelated packages).

This also means that when @vibe13 builds the official image, we may have no clue what packages get updated as part of that `yum update`. @vibe13, is this a problem for you?